applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py

```diff
@@ -9,7 +9,16 @@
 # Other imports
 from KratosMultiphysics.CompressiblePotentialFlowApplication.potential_flow_analysis import PotentialFlowAnalysis
 import KratosMultiphysics.kratos_utilities as kratos_utilities
+
 import os
+if kratos_utilities.IsApplicationAvailable("HDF5Application"):
+    has_hdf5_application = True
+else:
+    has_hdf5_application = False
+if kratos_utilities.IsApplicationAvailable("StructuralMechanicsApplication"):
```

Comment:
this is deprecated, use `kratos_utilites.CheckIfApplicationAvailable("HDF5Application", "StructuralMechanicsApplication")`